### PR TITLE
Feature/output to chart

### PR DESCRIPTION
**Pull request by seb231**
Removed redundant data files, updated valid-settings with groupings for Sankey plotting and removed from code, output ggplot Sankey plot to file for each year, created model parameter to stop acceptance test needlessly outputting files

### CODE REVIEW

File: src/clj/witan/send/charts.clj [hunk 1]
```diff
@@ -30,57 +31,6 @@
...
-            :PRU "Other"
-            :OOE "Other"
-            :FEC "Further education"
-            :ISC "Special"
-            :ISCR "Special"
-            :IT "Other"
-            :NON_SEND "Other"))
-
 (defn sankey [{:keys [title] :or {title ""}} df]
   (gg4clj/render
```
Comment (elisehuard) [inline src/clj/witan/send/charts.clj]: interesting use of gg4clj - wasn't expecting it out of a gorilla form.
maybe we should extract the non-gorilla bits (ie running the r code and outputting to a file) into a util

File: src/clj/witan/send/send.clj [hunk 2]
```diff
@@ -334,80 +341,93 @@
...
+        (let [columns [:calendar-year :ay-group :mean :std-dev :iqr :min :low-ci :q1 :median :q3 :high-ci :max]]
+          (->> (mapcat (fn [output year]
+                         (map (fn [[ay-group stats]]
+                                (-> (medley/map-vals round stats)
+                                    (assoc :ay-group ay-group :calendar-year year)))
+                              (:total-in-send-by-ay-group output))) send-output (range initial-projection-year 3000))
+               (map (apply juxt columns))
+               (concat [(map name columns)])
+               (csv/write-csv writer))))
+      (run! #(ch/sankey-transitions transitions-data % valid-settings) years)))
```
Comment (elisehuard) [inline src/clj/witan/send/send.clj]: just to be sure I understand, this line and lines around 360 are the only changes in this block?

File: src/clj/witan/send/charts.clj (hunk 1 above)
Comment (seb231) [reply src/clj/witan/send/charts.clj]: I thought about this, but at least at the moment this Sankey function is so specific to its use here. In the future we could maybe recreate it with more args to add more flexibility

File: src/clj/witan/send/send.clj (hunk 2 above)
Comment (seb231) [reply src/clj/witan/send/send.clj]: Yes all of the the code output and the new plot file output have been put in an `if` so when running tests it's not creating additional files

File: src/clj/witan/send/charts.clj (hunk 1 above)
Comment (elisehuard) [reply src/clj/witan/send/charts.clj]: fair

**Pull request merged**